docs/general/ledger.md

```diff
@@ -33,179 +33,75 @@ Check [Ledger Polkadot App](https://github.com/Zondax/ledger-polkadot) specifica
-
-In order to turn on the interoperability with the Polkadot Ledger application, go to the "Settings"
-tab in Polkadot-JS Apps. Find the option for attaching Ledger devices and switch the option from the
-default "Do not attach Ledger devices" to "Attach Ledger via WebUSB". Be aware: if you are not
-seeing this it is because there is
-[no Ledger support](https://github.com/polkadot-js/apps/issues/3771) on FireFox.
-
-![Dropdown selector for allowing Ledger connections in PolkadotJS Settings](../assets/ledger.png)
+:::info Ledger Live should be off while using Ledger with Polkadot-JS Apps as it can interfere with normal operation.
 
-Click "Save" to keep your settings.
-
-Now when you go to the "Accounts" tab you will see a new button that says "Add Ledger". Ensure that
-your Ledger device is unlocked and you have navigated into the Polkadot application, then click this
-button.
+:::
 
-![Add Ledger button in PolkadotJS](../assets/ledger/query-ledger.png)
+You can import your Ledger account to [Polkadot Extension](https://polkadot.js.org/extension/) or to the [Polkadot JS UI](https://polkadot.js.org/apps/#/explorer). For instructions
+on how to import Ledger accounts to the Polkadot JS extension read through [this support article](https://support.polkadot.network/support/solutions/articles/65000175387-how-to-add-your-ledger-through-the-polkadot-extension), while if you want to import Ledger accounts to the Polkadot JS UI you can consult [this other article](https://support.polkadot.network/support/solutions/articles/65000170812-how-to-add-ledger-account-through-the-polkadot-js-ui).
 
-A popup will appear asking you to select an account and derivation path.
+#### Derivation paths
 
-![Picking an account and derivation path](../assets/ledger/add-account.png)
+When adding a Ledger account using the extension or the UI, you will be asked to select an `account type` and an `account index`. The first lets you select an account, while the second lets you pick a derivation path from that account - think of it like a formula from which child accounts are generated. When you are creating a Polkadot ledger account for the first time on Ledger Live with name `Polkadot 1`, this can be added to Polkadot JS using the 0/0 derivation path (i.e. account type = 0 and account index = 0). If then you add a second account called `Polkadot 2`, this will correspond to the 1/0 derivation path, and so on. We thus have multiple parent accounts that cam be viewed and used in both Ledger Live and Polkadot JS. Additionally, we can use Polkadot JS to created multiple children accounts from each parent account. For example, `Polkadot 1` with 0/0 derivation path can have child 0/1, 0/2, etc. that can be used within the UI. However, such children accounts cannot be used in Ledger Live, as it only scans through the parent accounts. So, keep in mind that on Ledger Live balances on the children accounts cannot be viewed, and you cannot use such accounts.
```

Comment:
Fixed typo `cam be viewed` to `can be viewed`

Rephrased the last sentence

```suggestion
When adding a Ledger account using the extension or the UI, you will be asked to select an `account type` and an `account index`. The first lets you select an account, while the second lets you pick a derivation path from that account - think of it like a formula from which child accounts are generated. When you are creating a Polkadot ledger account for the first time on Ledger Live with name `Polkadot 1`, this can be added to Polkadot JS using the 0/0 derivation path (i.e. account type = 0 and account index = 0). If then you add a second account called `Polkadot 2`, this will correspond to the 1/0 derivation path, and so on. We thus have multiple parent accounts that can be viewed and used in both Ledger Live and Polkadot JS. Additionally, we can use Polkadot-JS UI to created multiple children accounts from each parent account. For example, `Polkadot 1` with 0/0 derivation path can have child 0/1, 0/2, etc. that can be used within the UI. However, such children accounts cannot be used in Ledger Live, as it only scans through the parent accounts. So, keep in mind that the balances on the children accounts cannot be viewed, and you will not be able to transact with those accounts on Ledger Live.
```